pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -138,26 +138,34 @@ def _configure_swa_callbacks(self):
             self.trainer.callbacks = [StochasticWeightAveraging()] + self.trainer.callbacks
 
     def configure_progress_bar(self, refresh_rate=None, process_position=0):
```

Comment:
Can we add some comments to this function delineating the 3 steps I mentioned earlier, for readability?
something like: 
```
# if progress bar callback already exists return it
# if rich is available and no RR set return new RPB
# else return TQDM PB
```